content/sample/images.md

```diff
@@ -24,3 +24,127 @@ sub: Using nunjucks
   alt='Miriam presenting',
   sizes='card'
 ) }}
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/graph.jpg',
+    alt: 'sketched graph on paper'
+  }],
+  class='media-pull',
+  attrs={'style': '--rowspan: span 2'}
+) }}
+
+## `media-pull` will position an image next to the text on wide enough screens
+
+The `embed.figure` macro is the only one that has access to the outer element in
+order to add attributes or classes. The `media-pull` class can be added to pull
+the image to the left of the content at the `xlarge` screen size. Using a style
+attribute, you can define how many rows to span using the css custom property
+`--rowspan`.
+
+
+## Aligning images within the content areas
+
+If you want an image to float to a side of the text, you need to first wrap the
+image and text in a div with a class of `contain`.
+
+<div class="contain">
+{{ embed.img(
+  src='work/coachhub/browse-coach-sitemap.jpg',
+  alt='site map for browsing coaches',
+  attrs={
+    'class': 'align-left'
+  }
+) }}
+
+To align an image to the left of the text, add the class `align-left`.
+Images will only float left or right if the screen is wide enough.
+</div>
+
+<div class="contain">
+{{ embed.img(
+  src='work/coachhub/browse-coach-sitemap.jpg',
+  alt='site map for browsing coaches',
+  attrs={
+    'class': 'align-right'
+  }
+) }}
+
+To align an image to the right of the text, add the class `align-right`.
+Images will only float left or right if the screen is wide enough.
+</div>
+
+
+<div class="contain">
+{{ embed.img(
+  src='work/coachhub/browse-coach-sitemap.jpg',
+  alt='site map for browsing coaches',
+  attrs={
+    'class': 'align-center'
+  }
+) }}
+
+To center align an image add the class `align-center`. This is only necessary
+if your image is smaller than the column width.
+
+</div>
+
+## Extending images outside the content column
+
+To extend an image further outside of the content column's width, add one of
+the following classes: `extend-large`, or `extend-full`. This is only available
+with the `embed.figure` macro as the `embed.img` macro's class doesn't have
+access to the outer container. Also, the embed.figure macro is wider than the
+content column by default, as opposed to the `embed.img` macro.
+
+### `embed.figure` without any entend class for comparison
```

Comment:
```suggestion
### `embed.figure` without any extend class for comparison
```

Seems like this could also be called `extend-small` - and available to other media?